lib/iris/fileformats/grib/__init__.py

```diff
@@ -984,7 +1049,7 @@ def as_messages(cube):
                       list of cubes.
```

Comment:
I thought we were going to deprecate "as_messages", in favour of just using the one-liner it contains ?